test/uk/gov/hmrc/plasticpackagingtax/registration/views/RegistrationViewSpec.scala

```diff
@@ -244,12 +245,14 @@ class RegistrationViewSpec extends UnitViewSpec with Matchers {
 
       "All Sections completed" when {
 
-        val spyCompleteRegistration = Mockito.spy(aRegistration())
-        when(spyCompleteRegistration.checkAndSubmitStatus).thenReturn(TaskStatus.Completed)
-        when(spyCompleteRegistration.isRegistrationComplete).thenReturn(true)
+        val completeRegistration = Mockito.spy(
```

Comment:
This doesn't need to be a mock, does it?

Reply:
no it doesn't

Reply:
this has been addressed